_pages/cases.md

```diff
@@ -57,12 +57,12 @@ lead: |-
-- [Utah Department of Corrections](https://www.justice.gov/crt/case/utah-department-corrections) On March 12, 2024, the Justice Department issued a letter notifying UDOC that it violated the ADA by discriminating against an incarcerated transgender woman on the basis of her disability, gender dysphoria. The letter outlined the minimum remedial measures UDOC must take to address the violations identified and invited UDOC to work with the United States to reach an appropriate resolution. [Learn More](https://www.justice.gov/crt/case/utah-department-corrections)
-- [U.S. v. MedStar Health, Inc.](https://www.justice.gov/crt/case/us-v-medstar-health-inc) On January 30, 2024, the United States filed a Complaint against MedStar Health, Inc., a leading healthcare provider in Maryland and the greater Washington, D.C., region. The lawsuit, filed under Title III of the ADA, alleges that MedStar Health denied people with disabilities equal access to medical care by excluding their necessary support persons. The suit alleges that MedStar Health failed to modify visitor restrictions so that people with certain disabilities, which affected their ability to independently access medical care, could be accompanied by their support persons (such as a family member, companion, or aide). These included individuals with dementia, intellectual disabilities, or autism spectrum disorder whose disabilities prevented them from providing medical history or understanding medical directions. As a result, they were unable to receive equal care without the assistance of their support person. [Learn More](https://www.justice.gov/crt/case/us-v-medstar-health-inc)
+- [In Re Georgia Senate Bill 202](https://www.justice.gov/crt/case/re-georgia-senate-bill-202) On January 31, 2024, the Justice Department filed a statement of interest explaining how the ADA's equal opportunity and reasonable modification requirements apply in the voting context. The statement of interest was filed in In Re Georgia SB 202, a consolidated lawsuit challenging restrictions on absentee and in-person voting under Georgia Senate Bill SB 202 (“SB 202”). One of the private lawsuits alleges that SB 202 fails to provide people with disabilities an equal opportunity to vote absentee and in-person or make reasonable modifications to avoid disability-based discrimination, in violation of Title II of the Americans with Disabilities Act and Section 504 of the Rehabilitation Act. The department’s brief explains that, under the ADA, voters with disabilities must have an equal opportunity to vote by a particular method as do voters without disabilities, and that this equal opportunity requirement is separate from the requirement that public entities make reasonable modifications. [Learn more](https://www.justice.gov/crt/case/re-georgia-senate-bill-202)
```

Comment:
> The statement of interest was filed in In Re Georgia SB 202

Just making sure this isn't a mistake. Is "In Re Georgia" correct, or do we have "in" here twice?

Reply:
This is the name of the bill (In Re) plus this was taken directly from justice.gov